Demo/Src/main.c

```diff
@@ -269,4 +273,34 @@ void assert_failed(uint8_t *file, uint32_t line)
 }
 #endif  /* USE_FULL_ASSERT */
 
+/**
+    Wire up the `stdio` system call, so that `printf()`

```

Comment:
nit: `stdio` is not a system call, suggest `write(2)`

Reply:
Updated this to be accurate